business-central-parent/business-central-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -437,6 +437,7 @@ See http://www.w3.org/TR/SVG/intro.html#MIMEType. -->
     <web-resource-collection>
       <web-resource-name>rest controller</web-resource-name>
       <url-pattern>/rest/controller/*</url-pattern>
+      <url-pattern>/docs</url-pattern>
```

Comment:
this should be added to `<web-resource-name>console</web-resource-name>` section

Reply:
console's auth-constraint has a full role, but /rest/controller/ just need admin, rest-all, according to the issue https://issues.redhat.com/browse/RHPAM-3284.

Reply:
operations on endpoints would still be protected, but allowing `/docs` to be visible to other roles can still be valuable, for instance, developer role can check endpoints available.

Reply:
you are right, but if the developer access docs, and he also can not do an operation.  the issue wants to avoid the second login.

Reply:
dev would be already logged in, in that case it won't have permission to execute the operation but it won't ask to login again.

Reply:
Done